mailpoet/assets/js/src/automation/api/index.ts

```diff
@@ -15,6 +15,12 @@ export type ApiError = {
 };
 
 export const initializeApi = () => {
-  apiFetch.use(apiFetch.createRootURLMiddleware(apiUrl));
+  apiFetch.use((options, next) => {
```

Comment:
We use the search component of WooCommerce now for searching products and categories:
https://github.com/mailpoet/mailpoet/pull/5162/commits/792ef36e7bd893318a0fca4676cc83e6b334db06#diff-21f48913c311bceda3481eef0550b8886d99c8383c75864d48e8fd59746b27abR2

`<Search />` will do its own requests through `apiFetch`. Without this change the URL would be `wp-json/mailpoet/wc-analytics/...` instead of `wp-json/wc-analytics`.

Reply:
Thanks for the additional details.